src/components/editor_flexbox.tsx

```diff
@@ -1,84 +1,171 @@
+    try {
+      setMdxModule(generateTemplateView(templateText));
+      localStorage.setItem("templateText", templateText);
+      setHasError(false);
+    } catch {
+      setHasError(true);
+    }
+  };
+
+  let MDXComponent = mdxModule.default;
+
   return (
-    <div className="flex-container">
-        <div className="flex-child">
-          <div className="form-group" >
-            <label>textarea</label>
-              <textarea
-                className='form-control'
-                id="message"
-                name="message"
-                value={text}
-                onChange={handleChange}
-                style={{ height: 'auto', minHeight: '700px', resize: 'none' }}
-              />
-            <button disabled={!text}>Submit</button>
-          </div>
-        </div>
-        <div className="flex-child">
-            <p><Markdown children={text} options={{
-            overrides: {
-              StringInput: {
-                component: StringInput,
-              },
-              TextInput: {
-                component: TextInput,
-              },
-              USStateSelect: {
-                component: USStateSelectWrapper,
-              },
-              PhotoInput: {
-                component: PhotoInput,
-              },
-              DataInput: {
-                component: DateInputWrapper,
-              },
-              NumberInput: {
-                component: NumberInputWrapper,
-              },
-              Collapse: {
-                component: Collapsible,
-              },
-              PrintSection: {
-                component: PrintSection,
-              },
-              Select: {
-                component: SelectWrapper,
-              },
-              Figure: {
-                component: FigureWrapper,
-              }
-            },
-        }}/></p>
-        </div>
-    </div>
+    <StoreProvider dbName="template_editor" docId={"playground"}>
+      <StoreContext.Consumer>
+        {({ doc }) => {
+          return (
+            <div className="container" id="mdx-container">
+              <div className="flex-container">
+                <div className="flex-child">
+                  <div className="form-group">
+                    <textarea
+                      className="form-control"
+                      id="message"
+                      name="message"
+                      value={templateText}
+                      onChange={handleSetText}
+                      style={{
+                        height: "auto",
+                        minHeight: "700px",
+                        resize: "none",
+                      }}
+                    />
+                    <button type="submit" onClick={handleButtonClick}>
+                      Re-Render
+                    </button>
+                  </div>
+                </div>
+                <div className="flex-child">
+                  {/* {!hasError && <MDXContent components={components} doc={doc} />}
```

Comment:
Please either remove this commented out text or provide an explanation (in the comment) for why it is being kept.